guides/doc-Administering_Project/master.adoc

```diff
@@ -12,6 +12,16 @@ include::common/assembly_accessing-server.adoc[leveloffset=+1]
 
 include::common/modules/proc_starting-and-stopping-server.adoc[leveloffset=+1]
 
+ifdef::satellite[]
+// Cloning Satellite Server
+include::common/cloning_satellite_server.adoc[leveloffset=+1]
```

Comment:
Please move this file to `guides/common/modules` and rename it to `proc_cloning-project-server.adoc` (if it's a procedure), else use `con_`.